nntrainer/src/databuffer.cpp

```diff
@@ -116,9 +119,16 @@ void DataBuffer::UpdateData(buffer_type type, std::ifstream &file) {
 
           unsigned int id = rangeRandom(0, mark.size() - 1);
           I = mark[id];
+          if (I > max_test || I < 0)
```

Comment:
I < 0 will generate errors in code verification services. Do not check < 0 with unsigned variables.

Reply:
Fixed.! Thanks.